doppler/api.go

```diff
@@ -177,12 +178,13 @@ func (client APIClient) PerformRequest(req *http.Request, params []QueryParam) (
 				Err:        nil,
 				Message:    strings.Join(errResponse.Messages, "\n"),
 				RetryAfter: retryAfter,
+				Response:   response,
 			}
 		}
-		return nil, &APIError{Err: fmt.Errorf("%d status code; %d bytes", r.StatusCode, len(body)), Message: "Unable to load response"}
+		return nil, &APIError{Err: fmt.Errorf("0 status code; 0 bytes", r.StatusCode, len(body)), Message: "Unable to load response", Response: response}
```

Comment:
Accidental `%d` change here?

Reply:
Hm.. Yeah. I have no idea how that got introduced. Pretty sure I never manually edited that, hah! 🤷